packages/backend/src/nest/storage/storage.service.ts

```diff
@@ -611,15 +611,15 @@ export class StorageService extends EventEmitter {
       }
     }
     await repo.db.load()
-    const allEntries = this.getAllEventLogRawEntries(repo.db)
+    // const allEntries = this.getAllEventLogRawEntries(repo.db)
     await repo.db.close()
     await repo.db.drop()
-    const hashes = allEntries.map(e => CID.parse(e.hash))
-    const files = allEntries
-      .map(e => {
-        return e.payload.value.media
-      })
-      .filter(isDefined)
```

Comment:
These changes were just to clean up some seemingly useless stuff.  We weren't doing anything with those hashes/files so its just wasted work.